specification/deviceregistry/DeviceRegistry.Management/assets.tsp

```diff
@@ -90,7 +90,7 @@ model AssetProperties {
   softwareRevision?: string;
 
   @doc("Reference to the documentation.")
-  documentationUri?: string;
+  documentationUri?: url;
```

Comment:
Please revert, this is a breaking change. Type change requires a new API version.
The same change broke our APIs in the past: https://github.com/Azure/azure-rest-api-specs/pull/29319

Reply:
fixed